sections/Timers/CampaignTimer.time117.tsx

```diff
@@ -0,0 +1,43 @@
+import Button from "$store/components/ui/Button.tsx";
+import Countdown from "$store/islands/Countdown.tsx";
+
+export interface Props {
+  /**
+   * @format date-time
+   */
+  date: string;
+  text: string;
+  labelButton: string;
+  link: string;
+}
```

Comment:
faltou a prop de se está justificado a direita ou esquerda

Reply:
Todas as props são obrigatórias, pra funcionar com props default voce deve torna-las opcional e atribuir valores default caso não sejam passados
```suggestion
export interface Props {
  /**
   * @format date-time
   */
  date?: string;
  text?: string;
  labelButton?: string;
  link?: string;
}
```